app/services/ping.rb

```diff
@@ -47,6 +50,39 @@ def ping_database
       }
     end
 
+    def ping_cache
+      response = {}
+
+      if redis_cache_store?
+        redis = Rails.cache.redis
+        response['servers'] = {}
+
+        if redis.respond_to?(:nodes)
+          nodes = redis.nodes
+        else
+          nodes = [redis]
+        end
+
+        nodes.each_with_index do |node, index|
+          start = current_time
+
+          begin
+            node.ping
+            status = STATUS_OK
+          rescue Redis::CannotConnectError
+            status = STATUS_FAIL
+          ensure
+            response['servers'][index] = {
+              status: status,
+              duration_ms: duration_ms(start),
+            }
```

Comment:
this results in an Hash with key `index` and value "the data". is that intentional?